src/Routes.js

```diff
@@ -1,23 +1,30 @@
+//#region > Imports
 //> React
 // Contains all the functionality necessary to define React components
-import React from 'react';
+import React from "react";
 // DOM bindings for React Router
-import { Route, Switch } from 'react-router-dom';
+import { Route, Switch } from "react-router-dom";
 
 //> Components
 /**
  * HomePage: A basic template page
  */
+/*
 import {
   HomePage
 } from './components/pages';
+*/
```

Comment:
I understand that this is the base, but to keep it clean please remove all unused imports.